bcgov/mds
# Disabled NoW edit application delays in prod

# Main

- disabled the ability to edit now delay dates in production until a bug that interferes with reporting is fixed

# Other

- fixed listing errors on that file

# How to test
- Ensure the edit button is not visible in production
![Screen Shot 2022-02-02 at 2 53 03 PM](https://user-images.githubusercontent.com/25125247/152251212-c0fd27c8-e3c7-4597-84b6-136d51d397b5.png)
